src/server/controllers/bounty.controller.ts

```diff
@@ -155,12 +155,7 @@ export const getBountyEntriesHandler = async ({
           ...i,
           metadata: i.metadata as ImageMetaProps,
         })),
-      files: files
-        .filter((f) => f.entityId === entry.id)
-        .map((f) => ({
-          ...f,
-          metadata: f.metadata as BountyEntryFileMeta,
-        })),
+      fileCount: files.length,
```

Comment:
We might want to have a separate lightweight db query just to get the file count if we are not going to return any file data here